### PR TITLE
fix: feedback button overlapping with title

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- Apply 24px padding when it is on mobile.

Preview:
image

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1052 #done

### CODE REVIEW

**Comment by rebelchris** [comment]
@sshanzel In the design it actually states this becomes a context button action.
Screenshot 2023-02-09 at 07 18 39

**Review by rebelchris — CHANGES_REQUESTED** [review]
@sshanzel It's not inline with the mobile version actually, missed that in my initial review.

So apparently that feedback button and the invite button become part of the context menu.

**Comment by sshanzel** [comment]
> @sshanzel It's not inline with the mobile version actually, missed that in my initial review.
> 
> So apparently that feedback button and the invite button become part of the context menu.

Oh, sure, let's update those missing items as well in this PR then, since we're already moving the feedback button.

**Comment by rebelchris** [comment]
@sshanzel I raised a request about it on Slack, I personally don't feel that invite button is a good idea.
So maybe let's await that part.

**Comment by sshanzel** [comment]
> So maybe let's await that part.

Cool. I'll adjust the feedback only then. Thanks!

File: packages/shared/src/components/squads/SquadHeaderMenu.tsx [hunk 1]
```diff
@@ -59,34 +69,24 @@ export default function SquadHeaderMenu({
...
-        </Item>
-      ) : (
-        <Item className="typo-callout" onClick={onLeaveSquad}>
-          <span className="flex items-center w-full typo-callout">
-            <ExitIcon size="medium" secondary={false} className="mr-2" /> Leave
-            Squad
-          </span>
-        </Item>
-      )}
+      <ContextMenuItem
```
Comment (el) [inline packages/shared/src/components/squads/SquadHeaderMenu.tsx:84]: this should be hidden on non-mobile i think

**Review by el — CHANGES_REQUESTED** [review]
small change

File: packages/shared/src/components/squads/SquadHeaderMenu.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/squads/SquadHeaderMenu.tsx:84]: Agreed.

**Review by rebelchris — CHANGES_REQUESTED** [review]
Nice addition for this component.
Optional you could even have a array that loops the potential items (like we do in sidebar, but it's up to you)

**Review by rebelchris — APPROVED** [review]

**Comment by sshanzel** [comment]
> Optional you could even have a array that loops the potential items (like we do in sidebar, but it's up to you)

Oh, right. Let me do that to keep things consistent. Thanks!

**Comment by sshanzel** [comment]
@rebelchris have updated it. Next we can create an options menu component to just accept props and these items to make it easier in the future.

**Pull request merged**